tests/page/page-dispatchevent.spec.ts

```diff
@@ -191,3 +191,25 @@ it('should dispatch absolute device orientation event', async ({ page, server })
   expect(await page.evaluate('gamma')).toBe(30);
   expect(await page.evaluate('absolute')).toBeTruthy();
 });
+
+it('should dispatch device motion event', async ({ page, server, isAndroid }) => {
+  it.skip(isAndroid, 'DeviceOrientationEvent is only available in a secure context. While Androids loopback is not treated as secure.');
```

Comment:
Same reason for skipping as in this one: https://github.com/microsoft/playwright/pull/28077